pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -315,6 +324,25 @@ loop:
 		}
 	}
 
+	// if we have restricted-v2, and we're not allowed (this means restricted-v2 did not match) and the user cannot use restricted-v1
+	// then we should check to see if restricted-v1 would allow the pod.  If so, prepend a specific failure message.
+	userCannotUseForRestricted := sets.NewString(denied...).Has("restricted")
```

Comment:
mini nit: has/is are good indicators for a boolean value.

Reply:
> mini nit: has/is are good indicators for a boolean value.
`userCannotUseForRestricted` flows well in the `if` sentence below:
> if restrictedV2 exists and if the pod is not allowed and if the user cannot use restricted, then add this special message to the list of errors.

Reply:
Well that reads surprisingly well. I didn't see it in the context of the if-statement.